File: 프로그래머스/0/120956. 옹알이 （1）/옹알이 （1）.py
def solution(babbling):
    answer = 0
    a=['aya','ye','woo','ma']
    for cur in babbling:
        s=cur
        for cur2 in a:
            s=s.replace(cur2,' ')
        s=s.replace(' ','')
        if not s:
            answer+=1
    return answer

File: 프로그래머스/0/120956. 옹알이 （1）/옹알이 （1）.java
class Solution {
    static String c[]={"aya","ye","woo","ma"};
    static boolean go(String s)
    {
        for (int i=0;i<4;i++)
            s=s.replaceAll(c[i]," ");
        s=s.replaceAll(" ","");
        if (s.length()==0)
            return true;
        return false;
    }
    public int solution(String[] babbling) {
        int answer = 0;
        for (int i=0;i<babbling.length;i++)
        {
            if (go(babbling[i])==true)
                answer+=1;
        }
        return answer;
    }
}